ReadMe.md

```diff
@@ -166,6 +166,40 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 
 ## FAQs
 
+#### About OpenSource Projects
+
+ OpenSource Projects means anybody is free to use, study and modify your project for any purpose. The open source community provides a great opportunity for aspiring programmers to distinguish themselves; and by contributing to various projects, developers can improve their skills and get inspiration and support from like-minded people. But most importantly, they can prove that they can build fantastic experiences that people love. means anybody is free to use, study, modify, and distribute your project for any purpose.
```

Comment:
Looks good... Please do a grammar check and make it right.

Reply:
Okay sir